docs/용어_사전.md

```
@@ -0,0 +1,13 @@
+## 개요
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)